scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -34,6 +34,8 @@ def _iv(A, k, ncv, tol, which, v0, maxiter,
     if math.prod(A.shape) == 0:
         message = "`A` must not be empty."
         raise ValueError(message)
+    if len(A.shape) != 2:
+        raise ValueError("Only 2-D input is supported for `A` (a single matrix)")
```

Comment:
Is this tested by the change to `_A_ndim_msg = "Only 2-D input"` in `test_svds.py`?